pystiche_papers/gatys_et_al_2017/_data.py

```diff
@@ -27,7 +26,7 @@ def images() -> data.DownloadableImageCollection:
                 "https://github.com/leongatys/NeuralImageSynthesis/blob/cced0b978fe603569033b2c7f04460839e4d82c4/Images/ControlPaper/fig2_content.jpg?raw=true",
                 file="house_concept_tillamook.jpg",
                 license=license(
-                    "https://associateddesigns.com/sites/default/files/plan_images/main/craftsman_house_plan_tillamook_30-519-picart.jpg"
+                    "https://associateddesigns.com/sites/default/files/plan_images/main/craftsman_house_plan_tillamook_30-519-picart.jpg",
```

Comment:
Was the extra comma added by you or by the code formatter? Please remove it and see if `black` puts it back. If it doesn't we can keep it that way.